testbot.js

```diff
@@ -1,6 +1,14 @@
+// import all the requirements at the top for readability
+import { Client, GatewayIntentBits } from 'discord.js';
+import fetch from 'node-fetch';
 import sqlite3 from 'sqlite3';
+import dotenv from 'dotenv';
 import fs from 'fs';
 
+// get the environment variables from the .env file
+dotenv.config();
+const { token } = process.env;

```

Comment:
Read token from the .env file and define it at the top

Generally all global variables should be defined here